internal/pkg/utils/errors/writer.go

```diff
@@ -0,0 +1,201 @@
+package errors
+
+import (
+	"bufio"
+	"fmt"
+	"strings"
+)
+
+const (
+	Indent = "  "
+	Bullet = "- "
+)
+
+type Writer interface {
```

Comment:
Writer som komplet prepisal, funguje uz bez `regexp` a je to cele prehladnejsie.